src/components/Form/StyledForm.js

```diff
@@ -0,0 +1,139 @@
+import dayjs from 'dayjs';
```

Comment:
It is a bit confusing that the name of the file is `StyledForm`. Usually, all components starting with "Styled..." only contain styled-components. I would suggest renaming this file to `Form.js`. Afterwards you can move all styled-components concerning your form into a file called `StyledForm.js`, same as you have done with the `StyledButton`.